user/using-workspaces.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Using Workspaces (Beta)
+layout: en
+---
+
+# Workspaces (Beta)
+
+> Workspaces are currently in beta testing, and may change without a prior
+> notice.
+{: .beta}
+
+Workspaces allow jobs _within_ a build to share files.
+They are useful when you want to use build artifacts from a previous job;
+for example, you might create a cache that can be used in multiple jobs later.
+
+While any build can theoretically use workspaces, they are most useful when
```

Comment:
```suggestion
Theoretically, any build can use workspaces, but they are most useful when
```

Reply:
How about

> So you can clearly see when a workspace is created and used, we recommend using workspaces with [build stages](/user/build-stages), as shown in the [following](#configuration) [examples](#other-confugurations).

(fixing links as needed to example headings below)